scout/server/blueprints/clinvar/views.py

```diff
@@ -132,26 +132,30 @@ def clinvar_update_submission(institute_id, submission):
 
 
 @clinvar_bp.route("/<submission>/download/json/<clinvar_id>", methods=["GET"])
-def clinvar_download_json(submission, clinvar_id):
+def clinvar_download_json(submission: str, clinvar_id: str):
     """Download a json for a clinVar submission"""
+    filename = clinvar_id if clinvar_id != "None" else submission
```

Comment:
👍  But, it would help me with a little explanation for the unusual `"None"` string in the docstring above - I had to reread the template to see why.